packages/contracts-bedrock/test/libraries/SafeCall.t.sol

```diff
@@ -141,10 +156,24 @@ contract SafeCall_Test is Test {
 
         for (uint64 i = 15_200_000; i < 15_300_000; i++) {
             uint256 snapshot = vm.snapshot();
+            uint256 expected;
+
+            // Because forge coverage always runs with the optimizer disabled,
+            // if forge coverage is run before testing this with forge test or forge snapshot, forge clean should be
+            // run first so that it recompiles the contracts using the foundry.toml optimizer settings.
+            if (vm.isContext(VmSafe.ForgeContext.Coverage)) {
+                // 15_278_645 is the exact amount of gas required to make the safe call
```

Comment:
Can we add comments on how you figured out these numbers? That way when e.g. a solc version bump changes this and breaks the test, it's easy to fix

Reply:
fixed